src/test/resources/execute-script.json

```diff
@@ -0,0 +1,1184 @@
+            "amount" : 2
+          },
+          {
+            "tokenId" : "23c1801e2392018004f6807f00ca807fff3e01fc00d8e20100f6805f5a7facff",
+            "amount" : 2
+          },
+          {
+            "tokenId" : "d2ff907f1f807f0100b27f807f12bb7f00ff824e53c37380ff80ffc3fb010501",
+            "amount" : 2
+          },
+          {
+            "tokenId" : "ee7f3dec00008080fd00e56fae80748cd7ff36acf401803b00657ff57e2e0158",
+            "amount" : 2
+          },
+          {
+            "tokenId" : "7ff4019a0180017f7f8ba8808cff46664100d2807f30800165008438dc807f68",
+            "amount" : 2
+          },
+          {
+            "tokenId" : "53017c7c7001007f9a2cffdc7f011180ff7f6a010a7fff00ff27007f00ff80ff",
+            "amount" : 2
+          }
+        ],
+        "creationHeight" : 10,
+        "additionalRegisters" : {
+          "R4" : "0201",
+          "R5" : "08cd02a73151bd3c8864d6cf9c81fa288df7940ea828dd6ccf96c8437805076c851fa0",
+          "R6" : "0414",
+          "R7" : "0e0a0a0a0a0a0a0a0a0a0a0a",
+          "R8" : "0ece1f0202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020
```

Comment:
this section is critical for practical use cases, so it should be covered by tests.
It is better to put some values here.

Reply:
My understanding of Ergo data model is not enough to provide some alternative values here:(

Reply:
This corresponds to ContextExtension of ErgoLikeContext.
You can check usages of ContextExtension in existing tests to find typical usages.
The key is Byte integer and value is any value supported by DataJsonEncoder.

Reply:
Sorry, where I can find this tests? I searched for ContextExtension in the ergo project but didn't find any references to it in tests. Also I looked at tests in ergo/src/test/scala/org/ergoplatform/http/routes but didn't find something relevant.

Reply:
I investigated it a little bit more. Why do you say that this fragment corresponds to ContextExtension? It is part of ErgoTransactionOutput which is decoded by JsonCodes as ErgoBox. But I still failed to find any test for it.